src/renderers/shaders/ShaderLib/meshphysical.glsl.js

```diff
@@ -52,6 +58,12 @@ void main() {
 
 	vWorldPosition = worldPosition.xyz;
 
+#endif
+
+#ifdef USE_ALPHAHASH
+
+	vObjectPosition = transformed.xyz;
```

Comment:
Can the existing varying `vWorldPosition` be used, instead?

Also, what is the reason this is chunk is physical-material-specific?

Reply:
Yes, and only for convenience of demonstrating the feature. I'll add it to other material types if we think this PR might move forward!

Reply:
@WestLangley Correction – `vWorldPosition` does not work as well, for reasons mentioned in the paper and apparent after I tested this:

> Hashing world-space coordinates provides stable noise for static geometry, and our early tests used world-space coordinates. However, this fails on dynamic geometry. Object-space coordinates give stable hashes for skinned and rigid transforms and dynamic cameras.

Given that we want the hash anchored to skinned and morphed surfaces, perhaps vObjectPosition is not the right name? We simply want to write the `position` attribute to a varying, would `vPosition` be appropriate?

Reply:
Yes.